helm/dagster/schema/schema/charts/dagster/subschema/retention.py

```diff
@@ -22,6 +22,7 @@ class Retention(BaseModel):
     sensor: TickRetention
     schedule: TickRetention
     autoMaterialize: Optional[TickRetention]
+    backfill: Optional[TickRetention]  # TODO - should this be optional?
```

Comment:
was following auto materialize precedent in making this Optional, but i don't know the history of why it was set as Optional